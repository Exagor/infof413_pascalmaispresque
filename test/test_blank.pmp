begin
    daz    :=    5
    print    (daz)
    dzfezfe
    if              azfzafz
end