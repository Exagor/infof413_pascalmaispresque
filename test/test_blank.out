token: begin	lexical unit: BEG
token: daz	lexical unit: VARNAME
token: :=	lexical unit: ASSIGN
token: 5	lexical unit: NUMBER
token: print	lexical unit: PRINT
token: (	lexical unit: LPAREN
token: daz	lexical unit: VARNAME
token: )	lexical unit: RPAREN
token: dzfezfe	lexical unit: VARNAME
token: if	lexical unit: IF
token: azfzafz	lexical unit: VARNAME
token: end	lexical unit: END
Variables :
azfzafz   5
daz   2
dzfezfe   4
